packages/bpk-component-graphic-promo/src/BpkGraphicPromo.module.scss

```diff
@@ -0,0 +1,256 @@
+      }
+    }
+  }
+
+  @include breakpoint-portrait-large {
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-heading-4;
+    }
+
+    &__kicker {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-subheading;
+    }
+
+    &__headline {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-hero-4;
+    }
+
+    &__strapline {
+      @include bpk-subheading;
+    }
+  }
+
+  @include breakpoint-portrait-medium {
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-heading-3;
+    }
+
+    &__kicker {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-subheading;
+    }
+
+    &__headline {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-hero-5;
+    }
+
+    &__strapline {
+      @include bpk-body-longform;
+    }
+  }
+
+  @include breakpoint-portrait-small {
+    &__sponsor-label {
+      margin-bottom: bpk-spacing-sm();
+
+      @include bpk-label-1;
+    }
+
+    &__kicker {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-label-1;
+    }
+
+    &__headline {
+      margin-bottom: bpk-spacing-md();
+
+      @include bpk-heading-2;
+    }
+
+    &__strapline {
+      @include bpk-body-default;
+    }
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+
+    @include breakpoint-landscape {
+      justify-content: center;
+
+      &--start {
+        align-items: start;
+      }
+
+      &--center {
+        align-items: center;
+      }
+
+      &--end {
+        align-items: end;
+      }
+    }
+
+    @include breakpoint-portrait {
+      justify-content: space-between;
+
+      &--inverted {
+        flex-direction: column-reverse;
+      }
+    }
+
+    @include breakpoint-landscape-desktop {
+      height: pxtorem(400);
+
+      @include bpk-margin-leading(bpk-spacing-xxxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xxl(), false);
+    }
+
+    @include breakpoint-landscape-tablet {
+      height: pxtorem(400);
+
+      @include bpk-margin-leading(bpk-spacing-xxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xl(), false);
+    }
+
+    @include breakpoint-portrait-large {
+      height: pxtorem(933);
+      padding: bpk-spacing-xl();
+    }
+
+    @include breakpoint-portrait-medium {
+      height: pxtorem(682);
+      padding: bpk-spacing-xl();
+    }
+
+    @include breakpoint-portrait-small {
+      height: pxtorem(500);
+      padding: bpk-spacing-lg();
+    }
+  }
+
+  &__sponsor-content,
+  &__promo-content {
+    display: flex;
+    flex-direction: column;
+
+    @include breakpoint-landscape {
+      &--start {
+        align-items: start;
+        text-align: start;
+      }
+
+      &--center {
+        align-items: center;
+        text-align: center;
+      }
+
+      &--end {
+        align-items: start;
+        text-align: start;
+      }
+    }
+
+    @include breakpoint-landscape-desktop {
+      width: pxtorem(360);
```

Comment:
Are we able to just define the rem value instead of doing a conversion?

Might also be worth seeing if we can make this up based on existing spacing values or at least creating a local variable to define what the number is?

As looking at this it just looks like a magic number and might cause confusion to future contributors or maintainers.

If you need the maths to do px to rem conversion its `pixel value / 16 = rem value`

Reply:
Yes, that's what the function does. I just thought about putting the pixel value there, because it matches the numbers you see on Figma, to avoid confusion as to what is what. The function converts it to `rem` to match the rest of the codebase, which is what ends up in the generated CSS.

Reply:
>because it matches the numbers you see on Figma

This is fine but people when using this component won't also look at it in Figma library/people externally to Skyscanner won't have access to it either.

I would switch from trying to calculate it through a separate function as it could cause issues with some build systems and weird calculations in production especially as some browsers can have weird rendering behaviour across browsers

Reply:
This also falls into alignment of how we define values across the codebase so keeps it consistent for us 😄

Reply:
I changed this to use pre-calculated constants in `rem`.